test/e2e/case13_templatization_test.go

```diff
@@ -461,44 +461,6 @@ var _ = Describe("Test templatization", Ordered, func() {
 				return configmap
 			}, defaultTimeoutSeconds, 1).ShouldNot(BeNil())
 
-			By("Patch with invalid hub template")
```

Comment:
Why this is removed?

Reply:
This is now handled in the governance-policy-framework:
https://github.com/open-cluster-management-io/governance-policy-framework-addon/pull/136